Notify/notification.ts

```diff
@@ -0,0 +1,59 @@
+import { QueueClient } from "@azure/storage-queue";
+
+import * as TE from "fp-ts/TaskEither";
+
+import { FiscalCode, NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { pipe } from "fp-ts/lib/function";
+import {
+  KindEnum as NotifyKind,
+  NotifyMessage
+} from "../generated/notifications/NotifyMessage";
+import { NotificationMessageKindEnum } from "../generated/notifications/NotificationMessageKind";
+import { toHash } from "../utils/crypto";
+
+export const base64EncodeObject = (_: unknown): string =>
+  Buffer.from(JSON.stringify(_)).toString("base64");
+
+export type SendNotification = (
+  fiscalCode: FiscalCode,
+  messageId: NonEmptyString,
+  notificationTitle: string,
+  notificationBody: string
+) => TE.TaskEither<Error, void>;
+
+/**
+ *
+ * @param notificationQueueClient
+ * @returns
+ */
+export const sendNotification = (
+  notificationQueueClient: QueueClient
+): SendNotification => (
+  fiscalCode,
+  messageId,
+  notificationTitle,
+  notificationBody
+): ReturnType<SendNotification> =>
+  pipe(
+    {
+      installationId: toHash(fiscalCode) as NonEmptyString,
+      kind: NotifyKind[NotificationMessageKindEnum.Notify],
+      payload: {
+        message: notificationBody,
+        message_id: messageId,
+        title: notificationTitle
+      }
+    },
+    (notifyMessage: NotifyMessage) =>
+      TE.tryCatch(
+        () =>
+          notificationQueueClient.sendMessage(
+            base64EncodeObject(notifyMessage)
+          ),
+        (error: { readonly message?: string }) =>
```

Comment:
Is it necessary? Can we infer this type from sendMessage return type?

Reply:
What about this?
```
flow(E.toError, error =>
  Error(
    `Error while sending notify message to the queue [${error.message}]`
  )
)
```

Reply:
Done in [d645e45](https://github.com/pagopa/io-functions-service-messages/pull/14/commits/d645e4548ba1a8623eb131cfeefe49696e208085) and added tests